### DISCUSSION TITLE
Arm版4.6.23无法打开消息中心

### DISCUSSION THREAD

**Discussion by SANA1096** [ywmoyue/biliuwp-lite#432] [Q&A] [answered]
### Bug 描述

打开消息中心提示已记录一个错误，页面空白并且显示未检测到合适版本的microsoft edge webview2运行时

### 操作步骤

```plain text
如上
```

### 期望结果

问题得到修复

### 截图

![微信图片_20231210233009](https://github.com/ywmoyue/biliuwp-lite/assets/153289945/9606c4d3-7911-43e0-8aec-40a41682bd8a)

### 日志

[20231210.log](https://github.com/ywmoyue/biliuwp-lite/files/13628309/20231210.log)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment, marked as answer]
由于B站的页面繁多，不可能一个一个都为其适配页面，因此对于某些难度高/不常用/其他原因的页面会直接用webview的网页打开。
你是否在运行某种精简版系统，或者手动删除了windows自带的edge浏览器？
可以尝试下载并安装microsoft edge webview2: https://developer.microsoft.com/zh-cn/microsoft-edge/webview2#download-section

**Reply by SANA1096** [reply]
问题已经解决了，确实是我精简版系统的问题，感谢帮助
